website: add community governance, etc. (#2353)
lima-vm/lima: website/content/en/docs/Community/Governance/_index.md

  but they do not have any access to merge pull requests nor push commits.
  A Reviewer is considered to as a candidate of Committers.

See also the [Contributing](../contributing) page.

### Current maintainers

| Name               | Role      | GitHub ID (not Twitter ID)                     | GPG fingerprint                                                                          |
|--------------------|-----------|------------------------------------------------|------------------------------------------------------------------------------------------|
| Akihiro Suda       | Committer | [@AkihiroSuda](https://github.com/AkihiroSuda) | [C020 EA87 6CE4 E06C 7AB9  5AEF 4952 4C6F 9F63 8F1A](https://github.com/AkihiroSuda.gpg) |
| Jan Dubois         | Committer | [@jandubois](https://github.com/jandubois)     |                                                                                          |
| Anders F Björklund | Committer | [@afbjorklund](https://github.com/afbjorklund) |                                                                                          |

> | Balaji Vijayakumar | Committer | [@balajiv113](https://github.com/balajiv113)   |                                                                                          |

Author: @lima-vm/maintainers Please add your GPG fingerprints, so that you all can be eligible to be a release manager (see the "Release process" section)

Reviewer: Apparently it has been a while since I gave up on signing... `pub   rsa2048 2018-06-10 [SC] [expired: 2019-06-10]`

Reviewer 2: Similar for me: `pub   rsa4096 2017-03-23 [SC] [expired: 2023-04-14]`.

I assume we can create a new key and upload it, even if it is not signed by other people?

Author: You can just renew expired keys https://filipe.kiss.ink/renew-expired-gpg-key/

Reviewer 3: I have been using SSH signing key. I can create a new GPG key and upload it

Reviewer: Will try to find and renew, otherwise will replace it.

Author: Thanks, added

Reviewer: New key:
`5981 D2E8 4E4B 9197 95B3  2174 DC05 CAD2 E73B 0C92`

https://github.com/afbjorklund.gpg

Author: Thanks, added

| Akihiro Suda       | Committer | [@AkihiroSuda](https://github.com/AkihiroSuda) | [C020 EA87 6CE4 E06C 7AB9  5AEF 4952 4C6F 9F63 8F1A](https://github.com/AkihiroSuda.gpg) |
| Jan Dubois         | Committer | [@jandubois](https://github.com/jandubois)     |                                                                                          |
| Anders F Björklund | Committer | [@afbjorklund](https://github.com/afbjorklund) |                                                                                          |
| Balaji Vijayakumar | Committer | [@balajiv113](https://github.com/balajiv113)   |                                                                                          |

<!-- TODO: invite non-committer reviewers -->

### Addition and promotion of Maintainers
An active contributor to the project can be invited as a Reviewer,
and can be eventually promoted to a Committer after 2 months at least.

A contributor who have made significant contributions in quality and in quantity
can be also directly invited as a Committer.

> A proposal to add or promote a Maintainer must be approved by 2/3 of the Committers who vote within 7 days.

Reviewer 2: Should there be a quorum for any of the voting, or is a single vote enough if nobody else votes in the time period?

Author: Maybe it should need 2 approvals (including the proposer) at least?

Author: Added "Voting needs 2 approvals at least. The proposer can vote too."

> A proposal should happen as a GitHub pull request to the Maintainer list above.

Reviewer 2: How are new maintainers added to the CNCF list?

Author: With a PR like this
- https://github.com/cncf/foundation/pull/500

It is highly suggested to reach out to the Committers before submitting a pull request to check the will of the Committers.

### Removal and demotion of Maintainers
A Maintainer who do not show significant activities for 6 months, or, who have been violating the Code of Conduct,
may be demoted or removed from the project.

A proposal to demote or remove a Maintainer must be approved by 2/3 of the Committers (excluding the person in question) who vote within 14 days.

A proposal may happen as a GitHub pull request, or, as a private discussion in the case of removal of a harmful Maintainer.
It is highly suggested to reach out to the Committers before submitting a pull request to check the will of the Committers.

### Other decisions
Any decision that is not documented here can be made by the Committers.

> When a dispute happens across the Committers, it will be resolved through a majority vote within the Committers.

Reviewer 2: Is there a tie-breaker, or is any tie a failed vote?

Author: I guess it should be considered a failed vote?
Basically we should try to reach consensus

Author: Added "A tie should be considered as a failed vote."

## Release process

Eligibility to be a release manager:

> - MUST be an active Commiter

Reviewer 2: ```suggestion
- MUST be an active Committer
```

Author: fixed

---
title: Governance
weight: 10
---

<!-- The governance model is similar to https://github.com/containerd/project/blob/main/GOVERNANCE.md but simplified -->

## Code of Conduct
Lima follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).

## Maintainership
Lima is governed by Maintainers who are elected from active contributors.

As a [Cloud Native Computing Foundation](https://cncf.io/) project, Lima will keep its [vendor-neutrality](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).

### Roles
Maintainers consist of two roles:

- **Committer** (Full maintainership): Committers have full write accesses to repos under <https://github.com/lima-vm>.

>   Committers' commits should still be made via GitHub pull requests (except for urgent secuirity fixes), and should not be pushed directly.

Reviewer 2: ```suggestion
  Committers' commits should still be made via GitHub pull requests (except for urgent security fixes), and should not be pushed directly.
```

Author: Thanks, fixed

  Committers are also recognized as Maintainers in <https://github.com/cncf/foundation/blob/main/project-maintainers.csv>.

- **Reviewer** (Limited maintainership): Reviewers may moderate GitHub issues and pull requests (such as adding labels and cleaning up spams),
  but they do not have any access to merge pull requests nor push commits.

>   A Reviewer is considered as a candidate of Committers.

Reviewer 2: ```suggestion
  A Reviewer is considered as a candidate to become a Committer.
```

Author: Thanks, fixed
